packages/graphql-api/src/resolvers/login-with-password.js

```diff
@@ -1,3 +1,10 @@
 export const loginWithPassword = Accounts =>
-  (async (_, { user, password }) =>
-    await Accounts.loginWithPassword(user, password));
+  (async (_, { user, userInput, password }) => {
+    if (userInput) {
```

Comment:
look at https://github.com/js-accounts/graphql/blob/master/packages/graphql-client/src/graphql-client.js#L70

userInput will always be undefined